server/channels/store/sqlstore/group_store.go

```diff
@@ -362,7 +395,13 @@ func (s *SqlGroupStore) Delete(groupID string) (*model.Group, error) {
 
 func (s *SqlGroupStore) Restore(groupID string) (*model.Group, error) {
 	var group model.Group
-	if err := s.GetReplicaX().Get(&group, "SELECT * from UserGroups WHERE Id = ? AND DeleteAt != 0", groupID); err != nil {
+	builder := s.getQueryBuilder().
+		Select("*").
+		From("UserGroups").
+		Where(sq.Eq{"Id": groupID}).
+		Where(sq.NotEq{"DeleteAt": 0})
```

Comment:
nit: I generally prefer as following but nothing blocking :)
```suggestion
		Where(sq.And{
			sq.Eq{"Id": groupID},
			sq.NotEq{"DeleteAt": 0},
		})